core-api/src/main/java/com/optimizely/ab/odp/ODPEventManager.java

```diff
@@ -119,12 +119,19 @@ public void identifyUser(@Nullable String vuid, @Nullable String userId) {
     }
 
     public void sendEvent(ODPEvent event) {
+        event.setData(augmentCommonData(event.getData()));
```

Comment:
is there a reason why we move this before isDataValid check?

Reply:
So, we can add extra protection against "commonData" injected from clients (android-sdk, etc) as well.